monitor/filter_exception_decoder.py

```diff
@@ -100,7 +112,12 @@ def rx(self, text):
                 self.buffer = ""
             last = idx + 1
 
-            m = self.BACKTRACE_PATTERN.match(line)
+            strip_line = self.strip_ansi(line)
+            # sys.stderr.write(
```

Comment:
commented out code?

Reply:
commented lines 116-118?
I left it because I spent time to find way how to see in debug logs non-printable characters

Reply:
Well, I'm not a maintainer here, so I have no saying, but in general – in the places that I do have a saying ;) – I forbid leaving commented out code, most of the time ;)

Better to add a dedicated function for that, with a switch or not, or whatev, to be reusable. Or just remove it.

Reply:
Sounds reasonable
In the previous commit, I removed these lines, but after a while they were needed again.
By the way, have you knowledge how I can add a switch to `platformio.exe device monitor` command to get it in the "addon" script?

Reply:
> how I can add a switch to `platformio.exe device monitor` command to get it in the "addon" script?

Sorry, IDK